pkgs/applications/gis/qgis/ltr.nix

```
@@ -1,8 +1,12 @@
 { lib, makeWrapper, symlinkJoin
-, qgis-ltr-unwrapped, extraPythonPackages ? (ps: [ ])
+, extraPythonPackages ? (ps: [ ])
+, libsForQt5
 }:
 with lib;
 symlinkJoin rec {
+
+  qgis-ltr-unwrapped = libsForQt5.callPackage ./unwrapped-ltr.nix {  };
```

personal tastes, to avoid issues around  `$qgis-ltr-unwrapped` in build
```suggestion
let
  qgis-ltr-unwrapped = libsForQt5.callPackage ./unwrapped-ltr.nix {  };
symlinkJoin rec {
```